R/scan_data.R

```diff
@@ -32,56 +32,65 @@
 #'data_20 = scan_data(stations, -20)
 #'
 #'#40 inch
-#'data_20 = scan_data(stations, -40)
+#'data_40 = scan_data(stations, -40)
 #'}
 #'@export
 scan_data = function(stationTriplets, depth, start=Sys.Date()-as.difftime(1, units='days'), end=Sys.Date()){
 	
+  returns = data.frame()
+  stationChunks = split(stationTriplets, ceiling(seq_along(stationTriplets)/5))
```

Comment:
the new api doesn't allow such large requests to be made, so I broke this up into small chunks for it to succeed